package.json

```diff
@@ -54,6 +54,7 @@
     "eslint-plugin-import": "^2.20.2",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^3.1.3",
+    "ember-template-lint-printer-test": "link:./test/fixtures/ember-template-lint-printer-test",
```

Comment:
This seems odd, why do we need this?

Reply:
This was to test being able to make sure we can load printers from the node_modules directory.